modules/services/taskwarrior-sync.nix

```diff
@@ -0,0 +1,55 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+{
+  meta.maintainers = with maintainers; [ minijackson pacien ];
+
+  options.services.taskwarrior-sync = {
+    enable = mkEnableOption "Taskwarrior periodic sync";
+
+    frequency = mkOption {
+      type = types.str;
+      default = "*:0/5";
+      description = ''
+        How often to run <literal>taskwarrior sync</literal>.
+        This value is passed to the systemd timer configuration as the onCalendar option.
+        See
+        <citerefentry>
+          <refentrytitle>systemd.time</refentrytitle>
+          <manvolnum>7</manvolnum>
+        </citerefentry>
+        for more information about the format.
+      '';
+    };
+  };
+
+  config = let
+    cfg = config.services.taskwarrior-sync;
```

Comment:
Please move this to the top to match the other modules.

Reply:
Done and amended.